public/playersBetController.js

```diff
@@ -0,0 +1,147 @@
+"use strict";
+export default class playersBetController {
+  constructor(players) {
+    this.currentPlayerTurn = 0;
+    this.players = players;
+    this.activePlayer = this.players[this.currentPlayerTurn];
+    this.currentBet = 0;
+    this.playersStillBetting = false;
+
+    const controlsContainerEle = document.querySelector("#game");
+    this.controlsContainerEle = controlsContainerEle;
```

Comment:
The variable `controlsContainerEle` is redundant as it is not being used for anything else other than assigning the value into `controlsContainerEle`, therefore the assignment can be done directly like this:
```suggestion
    this.controlsContainerEle = document.querySelector("#game");
```